### DISCUSSION TITLE
Adding CLEAR-10 benchmark to Avalanche

### DISCUSSION THREAD

**Discussion by linzhiqiu** [ContinualAI/avalanche#964] [Feature Request] [answered]
CLEAR-10 website: https://clear-benchmark.github.io

[Continual LEArning on Real-World Imagery (CLEAR)](https://clear-benchmark.github.io/) is the first continual image classification benchmark dataset with a natural temporal evolution of visual concepts in the real world that spans a decade (2004-2014). 

I am the first author of this work -- in our work we also use Avalanche (such a great CL framework) to conduct our baseline experiments, and it is now time for us to contribute back to the community! I will assign myself to this feature request.

**Comment by AndreaCossu** [comment]
much appreciated, thanks @linzhiqiu ! Let us know if you need any help with the pull request :smile:

**Comment by linzhiqiu** [comment, marked as answer]
I finished it! Just submit a PR.
